blackbox/docs/sql/aggregation.txt

```diff
@@ -330,6 +330,54 @@ Example::
     Due to java double precision arithmetic it is possible that any two executions
     of the aggregation function on the same data produce slightly differing results.
 
+percentile
+==========
+
+The ``percentile`` aggregation function computes a `Percentile`_ over numeric
+non-null values in a column.
+
+Percentiles show the point at which a certain percentage of observed values occur. 
+For example, the 98th percentile is the value which is greater than 98% of the observed values.
+
+The function expects a single fraction or an array of fractions and a column name.
+When supplied a single fraction, the function will return a single value corresponding
```

Comment:
and that null fractions produce null results, as well as computing a percentile on a null/empty column?